README.md

```diff
@@ -0,0 +1,5 @@
+# Infraestructura Virtual 21-22
+
+## Descripción
+
+Hoy en día hay varias plataformas para ver series, cada una con un catálogo diferente. Es por esto que es difícil llevar la cuenta de qué series has terminado o cuál es el último capítulo que viste sin tener que recorrer las distintas plataformas. Con el fin de disponer de esta información de manera sencilla, se ha desarrollado un bot de telegram para organizar todas las series que has visto o que estás viendo, las series que tienes pendientes e incluso añadir valoraciones.
```

Comment:
Por favor, no incluyáis nada de implementación. Y en cuanto a la idea, tratad de pensar en algo que no sea sólo almacenar y presentar de una forma determinada.

Reply:
```suggestion
Hoy en día hay varias plataformas para ver series, cada una con un catálogo diferente. Es por esto que es difícil llevar la cuenta de qué series has terminado o cuál es el último capítulo que viste sin tener que recorrer las distintas plataformas. Con el fin de disponer de esta información de manera sencilla, se ha desarrollado una aplicación para realizar un seguimiento de las series que has visto o que estás viendo, las series que tienes pendientes e incluso añadir valoraciones apoyándose en el catálogo de los diferentes servicios de streaming.
```
Creo que así podría detallar un poco más mi idea

Reply:
El problema no es tanto el detalle, sino el hecho de que sigue siendo, más o menos, almacenar información y presentarla al usuario. Es una aplicación perfectamente válida, pero entre almacenar información y presentarla al usuario hace falta una cierta cantidad de lógica de negocio, y procesamiento de información, para hacerla algo que se pueda desarrollar más allá de lo académico y/o superfluo en esta asignatura.